src/entry/app.js

```diff
@@ -18,6 +18,8 @@ import {
     inappPrivacyPolicy,
 } from "./inapp";
 
+window.addEventListener("touchmove", (e) => e.preventDefault());
```

Comment:
@Bloblblobl do you think we need to so this with `touchstart` too?

https://developer.mozilla.org/en-US/docs/Web/API/Element/touchstart_event